differ/storage.go

```diff
@@ -597,7 +638,7 @@ func (storage DBStorage) CleanupNewReportsForOrganization(orgID types.OrgID, max
 //
 // The method return number of deleted records.
 func (storage DBStorage) CleanupOldReportsForOrganization(orgID types.OrgID, maxAge string) (int, error) {
```

Comment:
We don't have any way to call this one, right?

Reply:
nope AFAIK, it was prepared for weekly report functionality, where this cleanup was planned